YetAnotherGame/Assets/Scripts/Controllers/PlayerController.cs

```diff
@@ -10,16 +10,33 @@ public class PlayerController : MonoBehaviour
 
     private Vector3 moveDirection = Vector3.zero;
     private CharacterController controller;
+    
+    [SerializeField]
+    [Header("First point view camera position")]
+    private Transform fpv = null;
+
+    [SerializeField]
+    [Header("Third point view camera position")]
+    private Transform tpv = null;
```

Comment:
А тут thirdPersonTransform
```suggestion
    private Transform  thirdPersonViewTransform= null;
```

Reply:
ок, вернем как было изначально